lib/PluginFindRecord/PluginFindRecord.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import className from 'classnames';
+import { noop } from 'lodash';
+
+import { Button } from '@folio/stripes/components';
+
+import css from './PluginFindRecord.css';
+
+class PluginFindRecord extends React.Component {
```

Comment:
functional component with hooks?

Reply:
yeah, I know... Actually, I think it's a good start - to leverage the common shitty code, get stable plugins (and consistent), and then we can refactor only one place. removing any magic, that we don't need